packages/nuxt/test/scan-components.test.ts

```diff
@@ -230,3 +243,21 @@ it('components:scanComponents', async () => {
   }
   expect(scannedComponents).deep.eq(expectedComponents)
 })
+
+const tests: Array<[string, string[], string]> = [
+  ['WithClientOnlySetup', ['Client'], 'ClientWithClientOnlySetup'],
+  ['Item', ['Item', 'Holder', 'Item'], 'ItemHolderItem'],
+  ['ItemHolder', ['Item', 'Holder', 'Item'], 'ItemHolderItem'],
```

Comment:
Oh, I think I might have added a wrong test  case here... 🤔

Reply:
> Oh, I think I might have added a wrong test case here... 🤔

oh? 🤣, I was really a little confused when I modified it, I thought there were rules I didn't know. So what are the rules now?
These three are the output results of the use cases that failed to run the test before, I think only the second one is wrong
```
1.['ItemHolder', ['Item', 'Holder', 'Item'], 'ItemHolderItemHolder'],
2.['ItemHolderItem', ['Item', 'Holder', 'Item'], 'ItemHolderItemHolderItem'], // should be ItemHolderItem
3.['Item', ['Item', 'Item'], 'ItemItem'],
```

Reply:
```suggestion
  ['ItemHolder', ['Item', 'Holder', 'Item'], 'ItemHolderItemHolder'],
```